tests/run-make/dep-info-spaces/Makefile.foo

```diff
@@ -1,7 +0,0 @@
-LIB := $(shell $(RUSTC) --print file-names --crate-type=lib $(TMPDIR)/lib.rs)
-
-$(TMPDIR)/$(LIB):
-	$(RUSTC) --emit link,dep-info --crate-type=lib $(TMPDIR)/lib.rs
-	touch $(TMPDIR)/done
-
--include $(TMPDIR)/lib.d
```

Comment:
Actually I stand corrected, this doesn't even check `lib.d` is valid Makefile syntax, so I'm fine with what the test does as-is.